browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -367,13 +367,25 @@ function _treelistboxControl(parentContainer, data, builder) {
 			tr.tabIndex = 0;
 			_headerlistboxEntry(tr, data, data.entries[i], builder);
 		}
+
+		setTimeout(function () {
+			var firstSelected = tbody.querySelector('.ui-listview-entry.selected');
+			if (firstSelected)
+				table.scrollTop = firstSelected.offsetTop;
```

Comment:
for me it worked better with the scrollintoview
```diff
@@ -371,7 +371,7 @@ function _treelistboxControl(parentContainer, data, builder) {
 		setTimeout(function () {
 			var firstSelected = tbody.querySelector('.ui-listview-entry.selected');
 			if (firstSelected)
-				table.scrollTop = firstSelected.offsetTop;
+				firstSelected.scrollIntoView();
```

Reply:
I remember that we had issue with scrollIntoView moving whole iframe in some cases (leaving blank space above notebookbar) so I wanted to avoid it. But we can try.

Reply:
Unfortunately it does scroll whole app with scrollIntoView, I guess it will be even more visible on mobile :(

Reply:
I think it was timing issue on your end, I updated the patch with different approach.